Merging attributes page into Manipulating Elements

ConnorCartwright opened the pull request:
Hey guys @AurelioDeRosa @agcolom  

Following the comments made in [issue #642](https://github.com/jquery/learn.jquery.com/issues/642) by devs4u I’ve merged the attributes page into the manipulating elements page. 

I've added some line spacing and moved around the text a bit in the _Manipulating Attributes section_. I also added in a sentence from the old attributes page and a getter code example.

Let me know if you have an feedback or if there are any problems and I'll get on it asap :bowtie:

agcolom commented:
@ConnorCartwright Thank for the PR :-) Looks good to me. @AurelioDeRosa Does this look good to you?

AurelioDeRosa commented on page/using-jquery-core/manipulating-elements.md:
```diff
@@ -132,23 +132,33 @@ myList.append( myItems.join( "" ) );
 
-jQuery's attribute manipulation capabilities are extensive. Basic changes are simple, but the `.attr()` method also allows for more complex manipulations. It can either set an explicit value, or set a value using the return value of a function. When the function syntax is used, the function receives two arguments: the zero-based index of the element whose attribute is being changed, and the curre ...
+jQuery's attribute manipulation capabilities are extensive.
+Basic changes are simple, but the `.attr()` method also allows for more complex manipulations.
```
Probably this line and the next can be combined with the previous one. I'm not sure we need break lines in between.

agcolom replied:
I agree with @AurelioDeRosa on this one

AurelioDeRosa commented on page/using-jquery-core/manipulating-elements.md:
```diff
@@ -132,23 +132,33 @@ myList.append( myItems.join( "" ) );
+jQuery's attribute manipulation capabilities are extensive.
+Basic changes are simple, but the `.attr()` method also allows for more complex manipulations.
+
+The `.attr()` method acts as both a getter and a setter. As a setter, `.attr()` can accept either a key and a value, or an object containing one or more key/value pairs.
```
Maybe we can link back to the relevant page of the API?

agcolom replied:
@AurelioDeRosa good point! I agree that would be useful

AurelioDeRosa commented on page/using-jquery-core/attributes.md:
```diff
@@ -1,27 +0,0 @@
-<script>{
```
I think that merging the two pages is great but I'm a bit worried about broken links. I think this page should redirect to the "Manipulating Attributes" section of the other page. Any thoughts?

AurelioDeRosa commented:
I've added a few comments to improve the PR. Overall I'm excited to see that we have a new contributor. Thank you @connor!

ConnorCartwright commented:
Hey @AurelioDeRosa 

No problem, thank you for the feedback!

I've removed the line spacing between those two sentences and added a link to the `.attr()` api page on its first mention.

Would it be best to add the `attributes.md` file back in and give it a meta tag to redirect, something like:

`<meta http-equiv="refresh" content="0; url=http://learn.jquery.com/using-jquery-core/manipulating-elements/" />`

Or is there a simpler way I'm missing?

AurelioDeRosa commented:
I think we might have a process in place for these situations but I'd like to have our expert opinion @gnarf
